value/src/main/java/com/google/auto/value/processor/AutoValueishProcessor.java

```diff
@@ -932,8 +932,25 @@ private boolean isInAutoValuePackage(String className) {
   ImmutableList<String> copiedClassAnnotations(TypeElement type) {
     // Only copy annotations from a class if it has @AutoValue.CopyAnnotations.
     if (hasAnnotationMirror(type, COPY_ANNOTATIONS_NAME)) {
-      Set<String> excludedAnnotations =
-          union(getExcludedAnnotationClassNames(type), getAnnotationsMarkedWithInherited(type));
+      Set<String> excludedAnnotations = ImmutableSet.<String>builder()
+          .addAll(
+              union(getExcludedAnnotationClassNames(type), getAnnotationsMarkedWithInherited(type)))
```

Comment:
You don't need the `union` anymore since you're using a builder. Just `addAll(getExcluded...).addAll(getAnnotations...)`.